src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -70,7 +95,7 @@ static std::string getDeviceFullName() {
             brand_string += ch[j];
     }
 #else
-    brand_string = "Non Intel Architecture";
+# error "Unknown device architecture"
 #endif
     return brand_string;
```

Comment:
looks like we can revert changes in this file, because they are already in master